.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 script:
   - npm run build
-  - npm run test
```

Comment:
should we also remove jest and testing stuff from dependencies?

Reply:
I actually left it intentionally because I noticed that someone commented they liked that the app was ready for tests with Jest (from that medium) article. I thought it was a nice touch. I can remove if we don't want to have an opinion on testing libraries